.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,36 @@
+name: Bug Report
+description: Report any bugs you encounter.
+body:
+  - type: textarea
+    attributes:
+      label: What is the issue?
+      description: Insert a detailed description of what your issue is
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: What is the issue with?
+      options:
+        - Deployment
+        - Effis
+        - Todel
+        - Pandemonium
+        - Oprish
+
+  - type: checkboxes
+    attributes:
+      label: Checks
+      options:
+        - label: Have you checked the documentation?
+          required: true
+        - label: Have you checked other issues to make sure this is not a duplicate?
+          required: true
+
+  - type: textarea
+    attributes:
+      label: Minimum Reproducible Sample/Code
+      description: Insert any code or steps to help reproduce this issue.
```

Comment:
If this has a `.`, shouldn't the other descriptions have it too?

Reply:
IMO remove them all

Reply:
IMO delete this PR